operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/EventDispatcher.java

```diff
@@ -87,9 +81,7 @@ private PostExecutionControl handleDispatch(ExecutionScope<R> executionScope) {
     }
     Context<R> context =
         new DefaultContext<>(
-            eventSourceManager,
-            new EventList(executionScope.getEvents()),
-            executionScope.getRetryInfo());
+            new EventList(executionScope.getEvents()), executionScope.getRetryInfo());
```

Comment:
What are these, formatting changes?

Reply:
And actually, it's not a formatting change. The `eventSourceManager` parameter got removed.

Reply:
ahh ok, then its good! thx